Jenkinsfile

```diff
@@ -271,6 +271,17 @@ podTemplate(name: podName,
                         pipelineUtils.setStageEnvVars(currentStage)
                         // We always run, but don't always push to artifacts
                         env.PUSH_IMAGE = "false"
+                        // Check if we should wipe daily image dir
+                        env.numImageDirs = 3
+                        env.dailyImageDir = pipelineUtils.getDailyImageDir(numImageDirs)
+                        env.wipeBool = pipelineUtils.checkDailyImageDir(numImageDirs)
+                        if (env.wipeBool) {
+                            // Wipe image dir
+                            env.rsync_paths = "."
+                            env.rsync_from = "\$(mktemp -d)/"
```

Comment:
hmmm...not sure this is going to work...bash is not executed here. you would have to do the 'sh' and get the stdout and somehow parse the new dir name.

Reply:
Yea... a lot of finger crossing going on. Maybe I'll just add a .sh file to a container to do all this for me and just call that